tests/PHPStan/Rules/Functions/CallToFunctionParametersRuleTest.php

```diff
@@ -411,6 +411,28 @@ public function testUnpackOperator(): void
 		]);
 	}
 
+	public function testFputCsv(): void
+	{
+		$this->analyse([__DIR__ . '/data/fputcsv-fields-parameter.php'], [
+			[
+				'Parameter #2 $fields of function fputcsv expects array<int|string, bool|float|int|string|null>, array<int, Person> given.',
+				33,
+			],
+		]);
+	}
+
+
+	public function testPutCsvWithStringable(): void
+	{
+		if (PHP_VERSION_ID < 80000) {
+			$this->markTestSkipped('Test skipped under 8.0');
```

Comment:
I read "under 8.0" as "when running 8.0". Something like "before" or "on lower version than" would be nicer.

Reply:
Added a better message.